### DISCUSSION TITLE
Gameplay freezes without fail after a duration after selecting an amount of objects in the editor (Lazer)

### DISCUSSION THREAD

**Discussion by NadosCS** [ppy/osu#24920] [Q&A]
### Type

Performance

### Bug description

Selecting an amount of objects in the editor of a map, exiting the editor and then playing any beatmap seems to produce a reliable freeze after roughly the same duration depending on the amount of objects selected. There seems to be a trend where the more objects you select, the sooner the freeze will come. 

I tried selecting objects in the editor from a number of maps, and to give an example, selecting ~2500 objects and then playing a map, will result in a freeze roughly 1 minute and 20 seconds into the beatmap. This may vary depending on the hardware, but it seems to be consistent on mine.

I noticed some other unusual performance related issues like smaller freezes, stutters etc. after i did this, however the big consistent freeze only seems to happen on the first beatmap you play after selecting the objects in the editor. I cant say for sure if the other performance related issues are related to this, but given i have not noticed them this often normally i assume they are.

Hope this helps in some way!

### Screenshots or videos

https://github.com/ppy/osu/assets/41868333/5deb39db-45b0-458d-b1a5-6433a63e0447

### Version

2023.914.0-lazer

### Logs

[legacy-ipc.log](https://github.com/ppy/osu/files/12705765/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/12705766/network.log)
[performance.log](https://github.com/ppy/osu/files/12705767/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12705768/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12705769/updater.log)
[database.log](https://github.com/ppy/osu/files/12705770/database.log)
[input.log](https://github.com/ppy/osu/files/12705771/input.log)

**Comment by bdach** [comment]
I can't really seem to replicate this and I can't confirm without frame graph showing (reupload of the video with it shown - ctrl-f11 twice - would help) but it kind of just looks like a regular GC pause. Not sure how worth it is to track this issue separately given https://github.com/ppy/osu/issues/17712 already exists.

**Comment by NadosCS** [comment]
I see, decided id report it on the off chance it would lead to something, my bad if i just wasted your time haha.

https://github.com/ppy/osu/assets/41868333/a53708b8-c247-4533-a34b-35cb00bc9cca
